src/workerd/api/web-socket.h

```diff
@@ -422,6 +425,8 @@ class WebSocket: public EventTarget {
     });
     JSG_TS_OVERRIDE(extends EventTarget<WebSocketEventMap>);
   }
+  kj::Maybe<kj::Date> autoResponseTimestamp;
```

Comment:
Mind putting this a little lower, under the `private` by `serializedAttachment`?